package.json

```diff
@@ -4,25 +4,28 @@
   "private": true,
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^9.1.2",
-    "@rjsf/core": "^5.5.2",
-    "@rjsf/utils": "^5.0.0-beta.2",
-    "@rjsf/validator-ajv8": "^5.5.2",
-    "@testing-library/jest-dom": "^5.16.5",
+    "@rjsf/core": "^5.13.0",
+    "@rjsf/material-ui": "^5.13.0",
+    "@rjsf/utils": "^5.13.0",
+    "@rjsf/validator-ajv8": "^5.13.0",
+    "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "ajv": "^8.11.0",
+    "ajv": "^8.12.0",
     "bootstrap": "^3.4.1",
-    "bootstrap-loader": "^4.0.0",
+    "bootstrap-loader": "^4.0.4",
     "bootstrap-sass": "^3.4.3",
     "file-saver": "^2.0.5",
-    "jquery": "^3.6.1",
-    "react": "^18.2.0",
-    "react-bootstrap": "^2.5.0",
+    "jquery": "^3.7.1",
+    "react-bootstrap": "^2.8.0",
     "react-datetime": "^3.2.0",
-    "react-dom": "^18.2.0",
-    "react-router-dom": "^6.3.0",
+    "react-router-dom": "^6.16.0",
     "web-vitals": "^2.1.4"
   },
+  "overrides": {
```

Comment:
Can you check if this necessary since you're upgrading rjsf everywhere? I added it to resolve a few dependency conflicts, but it looks like it might build without the overrides block.